api/app/scripts/update_wiki.py

```
@@ -0,0 +1,78 @@
+        output_file = os.path.join(wiki_data_dir, "processed_wiki.jsonl")
+
+        # Step 1: Download latest wiki dump
+        logger.info("Step 1: Downloading latest Bisq MediaWiki dump...")
+        download_main(output_dir=wiki_data_dir)
+        logger.info("Wiki dump downloaded successfully")
+
+        # Step 2: Process the dump
+        logger.info("Step 2: Processing wiki dump...")
+        if not os.path.exists(input_file):
+            logger.error(f"Input file not found after download: {input_file}")
+            raise FileNotFoundError(f"Wiki dump file not found: {input_file}")
```

Ruff (0.14.4)

Abstract `raise` to an inner function

(TRY301)